wordpress-mobile/gutenberg-mobile
# Initialize E2E tests with initial HTML

**Related PRs:**
* https://github.com/WordPress/gutenberg/pull/53434

Replaces the mechanism to set the initial HTML for E2E tests. Now, instead of switching to the HTML mode and pasting the content, we pass the initial HTML when starting the app.

**To test:**
* Observe that E2E tests pass in CI jobs (we need to run the optional tests).

PR submission checklist:
